gems/sorbet-runtime/lib/types/types/typed_array.rb

```diff
@@ -13,10 +13,15 @@ def underlying_class
     end
 
     # @override Base
-    def valid?(obj)
+    def recursively_valid?(obj)
       obj.is_a?(Array) && super
```

Comment:
```suggestion
      valid?(obj) && super
```

Reply:
^ This might be a change you want to make in a few other places too (just for consistency)